### PR TITLE
Validation: description-missing fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
Added missing descriptions to fix validation errors.

### CODE REVIEW

**Comment by brittmsantos** [comment]
@stevemunk @rbrundritt

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/v8-web-control/map-control-concepts/pushpins/default-pushpin-with-text-example.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Default Pushpin with Text Example | Microsoft Docs"
+description: Provides a code example that shows how to change the default pushpin on the map to have a number and a text label.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/pushpins/default-pushpin-with-text-example.md:3]:
```suggestion
description: Example code demonstrating how to create a pushpin at the center of the map and set values for the text, title and subTitle properties.
```

File: BingMaps/v8-web-control/map-control-concepts/pushpins/dynamically-create-svg-pushpin-example.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Dynamically create SVG Pushpin Example | Microsoft Docs"
+description: Provides a code example that shows how to dynamically create SVG circles for pushpin icons using a custom function.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/pushpins/dynamically-create-svg-pushpin-example.md:3]:
```suggestion
description: Example code demonstrating how to dynamically create SVG circles for pushpin icons using a custom function.
```

File: BingMaps/v8-web-control/map-control-concepts/pushpins/font-based-pushpins.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Font based Pushpins | Microsoft Docs"
+description: Provides how to use a custom font from the Font Awesome library to create custom pushpin icons for usage on a map.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/pushpins/font-based-pushpins.md:3]:
```suggestion
description: Example code demonstrating how to use a custom font to create a pushpin icon using an off-screen canvas to measure the text and create an image from the font. Fonts are created using the Font Awesome library.
```

File: BingMaps/v8-web-control/map-control-concepts/pushpins/image-and-canvas-pushpin-example.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Image and Canvas Pushpin Example | Microsoft Docs"
+description: Provides a code example that shows how to create custom colored pushpins that use an image as a template.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/pushpins/image-and-canvas-pushpin-example.md:3]:
```suggestion
description: Example code demonstrating how to create custom colored pushpins that use an image as a template.
```

File: BingMaps/v8-web-control/map-control-concepts/pushpins/index.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Pushpins | Microsoft Docs"
+description: Describes pushpins, which act as graphical image and text representations of locations on maps, and provides links to code examples.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/pushpins/index.md:3]:
```suggestion
description: The overview page for the Pushpins section contains a description and links to examples and related topics.
```

File: BingMaps/v8-web-control/map-control-concepts/rest-services-examples/index.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "REST Services Examples | Microsoft Docs"
+description: Describes the functionalities of the Bing Maps REST API services, and provides a list of Bing Maps REST services code examples.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/rest-services-examples/index.md:3]:
```suggestion
description: Describes the Bing Maps REST services, accessed using GET requests with JSONP, with example code demonstrating how to implement using standard JavaScript, jQuery, AngularJS.
```

File: BingMaps/v8-web-control/map-control-concepts/search-module-examples/index.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Search Module Examples | Microsoft Docs"
+description: Describes the Search module and provides a list of code examples and related topic articles to the Search module.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/search-module-examples/index.md:3]:
```suggestion
description: Describes the Search module and provides links to code examples and related topic. 
```

File: BingMaps/v8-web-control/map-control-concepts/search-module-examples/user-input-geocode-example.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "User Input Geocode Example | Microsoft Docs"
+description: Provides a code example that displays a search field and button which allows a user to perform a search for a location and then lists the results.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/search-module-examples/user-input-geocode-example.md:3]:
```suggestion
description: Example code demonstrating how to display a search field and button on a map. When the button is selected the search query is geocoded and the results are displayed on the map and a list of the result names are shown beside the map.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by brittmsantos** [comment]
@stevemunk 

I've implemented the requested changes. Please let me know if there are any other issues preventing approval and merge. Thank you, and have a great day!

**Pull request merged**